### ISSUE TITLE
Project S - column issue

### ISSUE THREAD

**Issue by xinyil** [Princeton-CDH/cdh-web#42]
<img width="463" alt="screen shot 2017-09-19 at 4 12 47 pm" src="https://user-images.githubusercontent.com/17734456/30613275-77abb26c-9d55-11e7-9971-5de5966b9e64.png">

In small view port, the it will only display one column of cards.

**Comment by rlskoeser** [comment]
Yes, I had that one working previously!  Not sure what changed, but I will track it down.  I meant to not mark the project mobile view ready for testing because I noticed the problem as I was looking through to see what you could review.
